tests/TestCase/Error/JsonApiExceptionRendererTest.php

```diff
@@ -5,31 +5,34 @@
 
 use Cake\Controller\Controller;
 use Cake\Core\Configure;
-use Cake\Core\Exception\Exception;
+use Cake\Core\Exception\CakeException;
 use Cake\Core\Plugin;
+use Cake\Datasource\ModelAwareTrait;
 use Cake\Http\Response;
 use Cake\Http\ServerRequest;
-use Cake\ORM\TableRegistry;
 use Crud\Error\Exception\ValidationException;
 use Crud\TestSuite\TestCase;
 use CrudJsonApi\Error\JsonApiExceptionRenderer;
 use Neomerx\JsonApi\Schema\ErrorCollection;
+use PHPUnit\Framework\MockObject\MockObject;
 
 class JsonApiExceptionRendererTest extends TestCase
 {
+    use ModelAwareTrait;
```

Comment:
This is not needed. `TestCase` already uses `LocatorAwareTrait` so you can use `$this->fetchTable()` to get table instances.